addon/components/basic-dropdown-trigger.ts

```diff
@@ -32,121 +27,4 @@ export default class BasicDropdownTrigger extends Component<Args> {
    * @memberof BasicDropdownContent
    */
   noop(): void {}
-
-  @action
-  handleMouseDown(e: MouseEvent): void {
-    if (this.args.dropdown.disabled) {
-      return;
-    }
-    if (this.args.eventType !== 'mousedown' || e.button !== 0) return;
-    if (this.args.stopPropagation) {
-      e.stopPropagation();
-    }
-    this._stopTextSelectionUntilMouseup();
```

Comment:
I removed this function. All it seemed to do was add & remove `ember-basic-dropdown-text-select-disabled` class name to the document body, a class name not referenced anywhere in this addon. I think it's just something from a long time ago

Reply:
(I can add it back if that causes problems)

Reply:
The reason for this class is so we can add `user-select: none;` to the document's body temporarily. This is to allow someone to mousedown on the trigger of a select, and without lifting its finger, move to the chosen option and lift then finder from the mouse. Since the dropdown is not physically (in the DOM) adjacent to the trigger, that flow tended to highlight the entire page.

I think we need it back.

Reply:
> The reason for this class is so we can add user-select: none; to the document's body temporarily

Good to know the reason - but as far as I can tell it's not doing that. There's no CSS for the `ember-basic-dropdown-text-select-disabled` class, and there's nothing that applies `user-select` style anywhere in this addon
https://github.com/cibernox/ember-basic-dropdown/search?q=ember-basic-dropdown-text-select-disabled&type=code
https://github.com/cibernox/ember-basic-dropdown/search?q=user-select&type=code

Or was this class intended for users of this addon to implement styles for?

Reply:
If memory serves, I never included default styles for this component but some other component i have based on this one (ember-power-datepicker maybe? It could be another) had styles targeting that class.

Reply:
Ah ok, will add back in. Looks like a lot of people definitely copied some styles from somewhere for it
https://github.com/search?l=CSS&q=ember-basic-dropdown-text-select-disabled&type=Code

Reply:
Added this back in to the component, rather than the modifier - was intending to keep the modifier without much frills or any external side effects (like adding classes to body element). Seems to work great